dev/devicelab/bin/tasks/commands_test.dart

```diff
@@ -13,10 +13,10 @@ import 'package:flutter_devicelab/framework/adb.dart';
 import 'package:flutter_devicelab/framework/framework.dart';
 import 'package:flutter_devicelab/framework/utils.dart';
 
-const int kObservatoryPort = 8888;
-
 void main() {
   task(() async {
+    final int vmServicePort = await findAvailablePort();
```

Comment:
This isn't a valid solution to this problem. By the time the port is re-opened by the process that we want to connect to, the port may have been opened by something else.

What we should do instead is have the target app open a port of its own choice and then connect to that based on what it reports in the console.